api/src/main/java/com/codeforcommunity/dto/ApiDto.java

```diff
@@ -100,6 +94,11 @@ protected boolean isEmpty(String str) {
    * @return a boolean representing whether the given password is invalid or not.
    */
   protected boolean passwordInvalid(String pass) {
-    return pass == null || pass.trim().length() < 8;
+    if (pass == null || pass.trim().isEmpty()) {
```

Comment:
Do the passwords still have to be 8 characters long?

Reply:
Yes, the regex on line `101` requires that they are between 8 and 20 characters

Reply:
What are the password requirements beyond that? I don't think we actually need anything else, but this Regex looks uhhhhhh complicated.